src/notifications/notificationmanager.cpp

```diff
@@ -103,6 +105,9 @@ NotificationManager::NotificationManager(QObject *parent) :
     databaseCommitTimer.setSingleShot(true);
     connect(&databaseCommitTimer, SIGNAL(timeout()), this, SLOT(commit()));
 
+    expirationTimer.setSingleShot(true);
+    connect(&expirationTimer, SIGNAL(timeout()), this, SLOT(expire()));
+
```

Comment:
Simo said it will just pause it, and this seems acceptable.  It will eventually expire, after n seconds of active usage.